ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -14,21 +14,13 @@ const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.AsphodelosTheThirdCircle,
   triggers: [
     {
-      id: 'P3N Experimental Fireplume 1',
+      id: 'P3N Experimental Fireplume Out',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ id: '6698', source: 'Protophénix', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ id: '6698', source: 'フェネクス', capture: false }),
-      response: Responses.getInThenOut(),
-    },
-    {
-      id: 'P3N Experimental Fireplume 2',
-      type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ id: '6696', source: 'Protophénix', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ id: '6696', source: 'フェネクス', capture: false }),
+      // This is Experimental Fireplume (6696) into Fireplume (6697), which starts casting 5s later.
```

Comment:
6696 is 11 seconds ahead of time.  6697 is 6 seconds ahead (which is maybe long too, but better).

Reply:
Given the size of the ground effect, I don't think it's unreasonable to have a 6-second warning. From the center, it's more than a GCD to get to safety, and for casters it can be impossible to get out if they have less time.

Reply:
I prefer the early call to slide cast into position.

Reply:
Ok, then maybe "Get out soon" instead of "Get out" which feels more immediate?

Reply:
Get out soon, although longer, would feel better. May want to have it leave the text up for extra time as it is early?

Reply:
I see it is "Out of middle soon" vs Get out soon", so the get out being shorter would be preferred.

Reply:
I think of "get out" as "get away from the boss" in this case it is "get away from the middle of the room" which feels different to me.  I added a duration, thanks for that suggestion.